packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -48,10 +51,19 @@ export const Alert: React.FunctionComponent<AlertProps> = ({
   title,
   children = '',
   className = '',
+  truncateTitle = 0,
   ouiaId,
   ouiaSafe = true,
   ...props
 }: AlertProps) => {
+  const titleRef = React.useRef(null);
```

Comment:
Unfortunately we can't use hooks in PF components due to older versions of React.

Reply:
No problem, I don't have to use Hooks for that. :+1:

Reply:
As of the breaking change release, we updated our React dependency to `react@^16.8.0` so we can now use hooks.